monitors/filterWords.js

```diff
@@ -0,0 +1,42 @@
+exports.conf = {
+  enabled: true,
+  // Change to false if the filter should not ignore other bots
+  ignoreBots: true,
+  // Change to true if the filter should ignore selfBots
+  ignoreSelf: false,
+};
+
+exports.run = (client, msg) => {
+  // Please add the channel ID you want to recieve these reports in.
+  const modlog = "CHANNELIDHERE";
+  if (msg.deletable) {
+    const badWords = msg.guild.settings.swearWords.filter(word => msg.content.toLowerCase().includes(word));
+    if (badWords.length > 0) {
+      let badWordList = [];
+      for (let i = 0; i < badWords.length; i++) {
+        badWordList += badWords[i];
```

Comment:
No, no, that's not how you push values into an array, you use [Array.prototype.push](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/push)